packages/fxa-auth-server/lib/senders/templates/layouts/multiSubscriptions.html

```diff
@@ -0,0 +1,345 @@
+<!DOCTYPE html>
```

Comment:
IMO, rather than a fully new layout, it could be better to use some [flags](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/email.js#L2084) and [conditionals](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/templates/layouts/subscription.html#L274) to switch content as needed in the footer. We do this for [the subscription cancellation email](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/email.js#L2038) to flip from "cancel" to "reactivate"